src/mon/FSCommands.h

```diff
@@ -47,6 +47,7 @@ class FileSystemCommandHandler : protected CommandHandler
       const int64_t pool_id,
       int type,
       bool force,
+      bool allow_overlay,
```

Comment:
For safety reasons, why can't we make `allow_overlay` default value `false`. So the caller can avoid passing `allow_overlay` value for types other than metadata?

Reply:
Make sense.
Updated.